packages/web/src/components/gcds-button/gcds-button.tsx

```diff
@@ -114,6 +114,11 @@ export class GcdsButton {
    */
   @Prop() download: string | undefined;
 
+  /**
+   * Whether the link is external or not
+   */
+  @Prop() external?: boolean = false;
```

Comment:
Because this is a prop change, I think it would be a good idea to be specific here since it only applies if the button is `type="link"`


```suggestion
  /**
   * Set to true if the button is meant to link to an external page. Requires property `type` to be set to `"link"` for this to work.
   */
  @Prop() externalLink?: boolean = false;
```

Reply:
Good idea!